pkg/certificate/mrc_reconciler.go

```diff
@@ -153,6 +159,33 @@ func (m *Manager) updateMRCComponentStatus(mrc *v1alpha2.MeshRootCertificate) er
 	return m.mrcClient.UpdateMeshRootCertificate(mrc)
 }
 
+// checks if the given MRC meets the attributes to move the componentStatuses to Validating
+func shouldChangeStatusToValidating(mrc *v1alpha2.MeshRootCertificate) bool {
+	// case-insensitive check for if strings are equal
+	if strings.EqualFold(string(mrc.Spec.Intent), constants.MRCIntentPassive) {
```

Comment:
can we prefer early returns. That is, instead of 

```go
if ... {
   if {
      ....
   }
}
return false
```

do this:

```go
if !... {
   return false
}
if ... {
  ...
}
```